.github/workflows/openstack/prune_openstack

```diff
@@ -1,84 +1,83 @@
-#!/usr/bin/env bash
+f#!/usr/bin/env perl
 
-export OS_AUTH_TYPE=v3applicationcredential
-export OS_AUTH_URL=https://keystone.hou-01.cloud.prod.cpanel.net:5000/v3
-export OS_IDENTITY_API_VERSION=3
-export OS_REGION_NAME="RegionOne"
-export OS_INTERFACE=public
+use Env;
```

Comment:
What are you using Env for? (please do not)

Reply:
Stale - yanked it out.